models/user.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+    toJSON() {
+      return { ...this.get(), id: undefined }
+    }
+  }
+  User.init({
+    uuid: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4
+    },
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      isAlpha: true
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      isAlpha: true
+    },
+    email: {
+      type: DataTypes.STRING,
+      isUnique: true,
+      allowNull: false,
+      unique: true,
+      validate: {
+        isEmail: true
+      }
+    },
+    phone: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      isNumeric: true,
+      unique: true
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      isAlphanumeric: true
+    },
+    token: {
+      type: DataTypes.TEXT ,
+      allowNull: true
+    },
+    tokenExpiration: {
+      type: DataTypes.INTEGER,
+      allowNull: true
+    },
+    isAdmin: {
```

Comment:
use _ for separation
eg:  is_admin current practice is correct but not recommended